packages/owl-bot/src/owl-bot.ts

```diff
@@ -150,6 +150,13 @@ export function OwlBot(
   // Run periodically, to ensure that up-to-date configuration is stored
   // for repositories:
   app.on('schedule.repository' as '*', async context => {
+    if (context.payload.syncLabels === true) {
+      // Sync labels.
+      const owner = context.payload.organization.login;
+      const repo = context.payload.repository.name;
+      await syncLabels(context.octokit, owner, repo, OWL_BOT_LABELS);
+      return;
```

Comment:
I don't understand all the contexts in which this function invoked.  I surprised to see a return statement here.  I would expect to see something like `if (context.payload.scanGithubForConfigs) {` before line 160.

Paging @bcoe who set up the original cron job.

Reply:
Added a parameter `scanGithubForConfigs`, also added some comment for clarification

Reply:
@SurferJeffAtGoogle Thanks, PTAL, and please let me know if the code is still unclear.

Reply:
I think what might make it clearer is if we made a `schedule.sync-labels` which would be run if the payload contains the `payload.syncLabels` standard field.

This could be follow up work, an we could track with issue?

---

With regards to having a `return` early, this is okay, there's no response sent during handler invocation, so an early return will just complete the promise.